utils/lavalog.go

```diff
@@ -152,26 +163,30 @@ func LavaFormatLog(description string, err error, attributes []Attribute, severi
 	return errRet
 }
 
+func LavaFormatPanic(description string, err error, attributes ...Attribute) {
+	attributes = append(attributes, Attribute{Key: "StackTrace", Value: debug.Stack()})
+	LavaFormatLog(description, err, attributes, LAVA_LOG_PANIC)
+}
+
 func LavaFormatFatal(description string, err error, attributes ...Attribute) {
 	attributes = append(attributes, Attribute{Key: "StackTrace", Value: debug.Stack()})
-	LavaFormatLog(description, err, attributes, 4)
-	os.Exit(1)
```

Comment:
is no one using this? someone in the protocol might rely on it killing the process?

Reply:
They don't - and they never did - because level 4 (Fatal) implies exit(1) by the logging library...